packages/cli/package.json

```diff
@@ -6,14 +6,13 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "bin": {
-    "expf": "./bin/expf.sh"
+    "expf": "./bin/expf.mjs"
```

Comment:
Just commenting here to connect the dots to the other comment. We need to keep entry scripts that allow specifying node options. We may not need something as complicated as what `npm` does, but we at least need to keep the shell script wrappers.

Reply:
All `npm` and the various package managers do is create links from `node_modules/.bin/expf -> node_modules/@expressjs/perf-cli/bin/expf.sh`. So it can be any type of executable.